docs/generate_submission_key.rst

```diff
@@ -56,36 +56,36 @@ Create the Key
 Export the *Submission Public Key*
 ----------------------------------
 
-Navigate to **Applications ▸ Utilities ▸ Passwords and Keys** to open a
-graphical interface to manage GPG keys. Click the clipboard icon |gpgApplet|
-in the top right corner and select "Manage Keys". Click "GnuPG keys" and you
-should see the key that you just generated.
+Navigate to **Applications ▸ Utilities ▸ Kleopatra** to open a
+graphical interface to manage GPG keys. At the bottom of the list
+of keys, you will find the SecureDrop Submission Key you just created.
```

Comment:
It showed up under the topmost item in the list for me, but that may be because multiple keys were already present in the keyring. To avoid any situation where this is inaccurate, I would just say "In the list of keys .." (or instruct them to sort the list of keys first).

Reply:
Thank you, I (incorrectly) assumed that new keys would be added to the bottom since that's what I've seen each time I've followed these steps. Just to avoid any discrepancies based on how users sort their list of keys, I'll change this to just say that the SD Submission Key can be found in the list.